users/src/main/proto/spine/sharing/sharing.proto

```diff
@@ -117,17 +117,30 @@ message Permissions {
 // An object can be shared to a wide audience (such as organization or all users of the system),
 // or only to specific people.
 //
+// The object can have own sharing settings or take it from an object into which it is enclosed (parent object).
+//
 // If a subject is not added to sharing settings of an object, the level is `AccessLevel.NONE`.
 //
 message SharingSettings {
-    oneof audience {
-        // All users of the system can access.
-        IndirectPermission public = 1;
 
-        // All organization members can access.
-        IndirectPermission organization = 2;
+    oneof mode {
```

Comment:
As discussed, we need to include both `take_from_parent` and `settings` simultaneously.

This will allow us to simulate Drive functionality by overriding the parent `SharingSettings` on a child object.